src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -438,6 +438,17 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP079",
                 "This expression is referencing its own declaration, which is not allowed.");
+
+            public ErrorDiagnostic SymbolicNameDoesNotExistWithSuggestion(string name, string suggestedName) => new ErrorDiagnostic(
+                TextSpan,
+                "BCP081",
```

Comment:
I just noticed that `BCP060` and `BCP61` are not used. I wonder if I should use them instead.

Reply:
Since we are still in "preview", it's fine either way. If we were already GA or supported in PROD, we should only increment the error codes. (Otherwise, there could be confusion with stale docs, blogs, SO threads, etc if we reuse a code.)

Reply:
> If we were already "GA" or supported in PROD, we should only increment the error codes
I see, this make sense. I'll leave it as is then.